log/context.go

```diff
@@ -50,8 +54,52 @@ const (
 
 	// JSONFormat represents the JSON logging format
 	JSONFormat = "json"
+
+	// TraceLevel level.
+	TraceLevel = logrus.TraceLevel
+
+	// DebugLevel level.
+	DebugLevel = logrus.DebugLevel
+
+	// InfoLevel level.
+	InfoLevel = logrus.InfoLevel
 )
 
+// SetLevel sets log level globally.
+func SetLevel(level string) error {
```

Comment:
Is it better if the type of the parameter is Level?  so that when using it, we can call `log.SetLevel(log.InfoLevel)`

Of course, we can also add a check for the level parameter, eg.
```go

// SetLevel sets log level globally.
func SetLevel(level Level) error {
        lvl, err := logrus.ParseLevel(level.String())
        if err != nil {
                return err
        }

        logrus.SetLevel(lvl)
        return nil
}